### PR TITLE
CAW21-76 Build sweetener content type and views

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Sweetener content type and started view.
- More styling and polishing coming later.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout `caw` branch of https://github.com/SU-SWS/ace-gryphon
2. `composer require SU-SWS/caw_profile:dev-CAW21-76 --no-update; composer update -n`
3. `blt drupal:sync --site=cardinalatwork`
4. `drush @cardinalatwork.local mim --group=default`
5. log in and view some of the sweetener content

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by joegl — APPROVED** [review]
This LGTM. Migration worked without issue (pulled 93 over). 

I did not get any images in the migrated content, however I have not been able to pull down any files on my local yet.

**Comment by joegl** [comment]
Images are working locally now 👍

File: config/sync/core.entity_view_display.node.stanford_news.default.yml [hunk 1]
```diff
@@ -163,7 +158,6 @@ third_party_settings:
                 view_mode: view_mode
             additional: {  }
             weight: 0
-            third_party_settings: {  }
```
[the remark this answers is not in the document]
Comment (pookmish) [reply config/sync/core.entity_view_display.node.stanford_news.default.yml:166]: Just updated contrib changes the configs. Nothing to worry about IMO

File: tests/codeception/acceptance/Users/RolesCest.php [hunk 2]
```diff
@@ -229,7 +229,7 @@ protected function runLinkExistCheck(AcceptanceTester $I, array $links, $can_see
         continue;
       }
 
-      $I->cantSeeLink($link_text, $path);
+      $I->cantSeeLink($link_text, $path ?? '');
```
[the remark this answers is not in the document]
Comment (pookmish) [reply tests/codeception/acceptance/Users/RolesCest.php:232]: very handy :D

File: config/sync/core.entity_view_display.node.stanford_news.default.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (joegl) [reply config/sync/core.entity_view_display.node.stanford_news.default.yml:166]: Oh yea not worried, I'm just used to seeing these and was curious what was dropping them.

**Pull request merged**